From d42cf5908d3581b4e52adc42bf034c752951bf35 Mon Sep 17 00:00:00 2001
From: Julien Cristau <jcristau@debian.org>
Date: Thu, 26 Nov 2009 16:32:00 +0100
Subject: [PATCH 5/5] config/udev: look for xkb{rules,model,layout,variant,options} instead of xkb.*

This makes the Debian/Ubuntu rule to generate this much simpler.
---
 config/udev.c          |    2 +-
 config/x11-input.rules |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/config/udev.c b/config/udev.c
index 62b9052..bbaf1fa 100644
--- a/config/udev.c
+++ b/config/udev.c
@@ -35,7 +35,7 @@
 #include "config-backends.h"
 #include "os.h"
 
-#define UDEV_XKB_PROP_KEY "xkb."
+#define UDEV_XKB_PROP_KEY "xkb"
 #define UDEV_PROP_KEY "x11_options."
 
 static struct udev_monitor *udev_monitor;
diff --git a/config/x11-input.rules b/config/x11-input.rules
index 59ef034..21e0e96 100644
--- a/config/x11-input.rules
+++ b/config/x11-input.rules
@@ -1,7 +1,7 @@
 SUBSYSTEM!="input", GOTO="x11_input_end"
 
 KERNEL=="event*", ENV{x11_driver}="evdev"
-KERNEL=="event*", ENV{ID_CLASS}=="kbd", ENV{xkb.layout}="us"
+KERNEL=="event*", ENV{ID_CLASS}=="kbd", ENV{xkblayout}="us"
 KERNEL=="event*", ENV{ID_CLASS}=="mouse", ATTRS{protocol}=="AlpsPS/2", ENV{x11_driver}="synaptics"
 
 LABEL="x11_input_end"
-- 
1.6.5

